src/main/java/net/imglib2/img/basictypeaccess/AccessFlags.java

```diff
@@ -0,0 +1,56 @@
+package net.imglib2.img.basictypeaccess;
+
+import net.imglib2.Dirty;
+import net.imglib2.img.basictypeaccess.array.ArrayDataAccess;
+import net.imglib2.img.basictypeaccess.volatiles.VolatileAccess;
+
+/**
+ * Flags that specify variants of {@link ArrayDataAccess} underlying primitive
+ * types. {@link #DIRTY} means that an access implements {@link Dirty}.
+ * {@link #VOLATILE} means that an access implements {@link VolatileAccess}.
+ *
+ * @author Tobias Pietzsch
+ */
+public enum AccessFlags
+{
+	DIRTY, VOLATILE;
```

Comment:
`AccessFlags` are only used for unused optional parameter in `ArrayDataAccessFactory`.
`imglib2-cache` is probably a better place to be for this class.